pkg/csi/service/vanilla/controller.go

```diff
@@ -197,8 +197,8 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 							log.Infof("Successfully reloaded configuration from: %q", cfgPath)
 							break
 						}
-						log.Errorf("failed to reload configuration. will retry again in 5 seconds. err: %+v", reloadConfigErr)
-						time.Sleep(5 * time.Second)
+						log.Errorf("failed to reload configuration. will retry again in 60 seconds. err: %+v", reloadConfigErr)
```

Comment:
Can we also add an attempt counter and crash CSI driver if the supplied password is incorrect?

Reply:
I though this was originally added to retry until login credentials are correct

Reply:
I am no longer changing this existing logic. I got to the root cause of the issue and with that, I did not see any incorrect login attempt by CSI